### PR TITLE
Add docs for string length scalar functions

### PR DESCRIPTION
**Pull request by kovrus**

### CODE REVIEW

File: docs/sql/scalar.txt [hunk 1]
```diff
@@ -101,6 +105,92 @@ which defines the length of the substring::
...
+-------------------------------------
+
+The ``char_length`` returns number of characaters in the string.
+
+synopsis
+........
+
+::
+
+    CHAR_LENGTH (str)
```
Comment (mfussenegger) [inline docs/sql/scalar.txt]: the other functions so far don't have a synopsis, do we really need one for the new ones?

File: docs/sql/scalar.txt [hunk 2]
```diff
@@ -101,6 +105,92 @@ which defines the length of the substring::
     SELECT 1 row in set (... sec)
 
 
+.. _scalar_char_length:
+
+char_length('string') returns integer
+-------------------------------------
+
+The ``char_length`` returns number of characaters in the string.
```
Comment (mfussenegger) [inline docs/sql/scalar.txt]: typo in characters

File: docs/sql/scalar.txt (hunk 1 above)
Comment (kovrus) [inline docs/sql/scalar.txt]: there are two functions which already have synopsis

**Comment by mfussenegger** [comment]
ack

**Pull request merged**